PythonVisualizations/Array.py

```diff
@@ -259,21 +247,14 @@ def search(self, item):
     return self.get(self.find(item))
     """
 
-    searchCodeSnippets = {
-        'call_find': ('2.20','2.35'),
-        'call_get': ('2.11','2.end'),
-        'return': ('2.4', '2.end'),
-    }
-
     def search(self, item):
         self.startAnimations()
-        callEnviron = self.createCallEnvironment(
-            self.searchCode.strip(), self.searchCodeSnippets)
-        self.highlightCodeTags('call_find', callEnviron)
+        callEnviron = self.createCallEnvironment(self.searchCode)
+        self.highlightCode('self.find(item)', callEnviron)
         n = self.find(item)

```

Comment:
The circle drawn by find disappears soon after and does not remain on the screen after the animation finishes. Is this intended? I could see the argument going either way

Reply:
That's a good point.  The oval was being erased when find() was popped off the stack.  I changed `search()` to recreate the oval in commit 1f358c0.  That makes the oval flash briefly, which is not ideal, but the oval remains when search() exits because the temporary variables of the bottom-most call on the stack stay visible.